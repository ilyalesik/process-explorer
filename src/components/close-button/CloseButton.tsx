import * as React from "react";
import * as s from "./closeButton.css";

export const CloseButton = props => {
	return <button {...props} className={s.closeButton} />;
};
