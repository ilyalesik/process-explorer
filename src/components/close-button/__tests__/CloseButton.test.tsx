import * as React from "react";
import * as renderer from "react-test-renderer";
import { CloseButton } from "../CloseButton";

describe("<CloseButton />", () => {
	it("CloseButton renders correctly", () => {
		const tree = renderer.create(<CloseButton onClick={() => {}} />).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
