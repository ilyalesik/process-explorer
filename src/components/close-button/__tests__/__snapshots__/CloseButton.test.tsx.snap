// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CloseButton /> CloseButton renders correctly 1`] = `
<button
  className="closeButton"
  onClick={[Function]}
/>
`;
